FileIO/XmlIO/Boost/BoostXmlGmlInterface.cpp

```diff
@@ -334,4 +337,71 @@ bool BoostXmlGmlInterface::write()
 	return true;
 }
 
+void BoostXmlGmlInterface::addSurfacesToPropertyTree(
+	BaseLib::ConfigTree & geometry_set)
+{
+	GeoLib::SurfaceVec const*const sfc_vec(_geo_objects.getSurfaceVecObj(_exportName));
+	if (!sfc_vec) {
+		INFO("BoostXmlGmlInterface::addSurfacesToPropertyTree(): "
+			"No surfaces within the geometry \"%s\".", _exportName.c_str());
+		return;
+	}
+
+	std::vector<GeoLib::Surface*> const*const surfaces(sfc_vec->getVector());
+	if (! surfaces) {
```

Comment:
This should not happen. If there are no surfaces in the vector `surfaces` then this should be a `nullptr`. However, I will insert an additional check to prevent errors in the future when the behaviour of `GEOObjects::getSurfaceVec()` or/and `SurfaceVec::getVector()` will be eventually changed, i.e., returns an empty vector instead of `nullptr`.